### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by mariordgez**
In this project, I build a simple HTML list of To Do tasks. The list is styled according to the specifications listed  in the lesson. This simple web page will be built using webpack and served by a webpack dev server.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,75 @@
...
+👤 **Mario Alberto Rodriguez Cota**
+
+- GitHub: [@mariordgez](https://github.com/mariordgez)
+- Twitter: [@MarioRo75396624](https://twitter.com/MarioRo75396624)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/mario-alberto-rodriguez-cota-a2860a205)
+
+## Cloning repo for non programmers
+
+1. On GitHub, navigate to the main page of the repository.
+
```
Comment (hemantso) [inline README.md]: - [ ]  In this getting started part there are some instructions that you have missed for a user to use your project. such as `npm install`, and `npm start`. And please try to follow [this](https://github.com/microverseinc/readme-template) design template.

File: src/index.js [hunk 2]
```diff
@@ -1,14 +1,57 @@
...
+
+const todoInput = document.querySelector('.todo-input');
+const todoAdd = document.querySelector('.todo-add');
+const todoList = document.querySelector('.todo-list');
+const task1 = new Task('hello', 0);
+const task2 = new Task('hello', 1);
+const taskArr = [task1, task2];
+
+function addToUI(task) {
+  const todoDiv = document.createElement('div');
```
Comment (hemantso) [inline src/index.js]: - [ ] As per the learning objective we have to follow the guidelines of ES6 syntax. Kindly try to use the arrow function.

File: src/index.js [hunk 3]
```diff
@@ -1,14 +1,57 @@
...
+  taskArr.push(newTask);
+  console.log(taskArr);
+  addToUI(newTask);
 }
 
-document.body.appendChild(component());
+todoAdd.addEventListener('click', todo);
+
+function uiList(arr) {
+  arr.forEach((task) => {
```
Comment (hemantso) [inline src/index.js]: - [ ] As per the learning objective we have to follow the guidelines of ES6 syntax. Kindly try to use the arrow function.

File: src/index.js [hunk 4]
```diff
@@ -1,14 +1,57 @@
...
-  element.classList.add('hello');
+  const todoDots = document.createElement('button');
+  todoDots.classList.add('todo-btn');
+  todoDots.innerHTML = '<i class="fas fa-ellipsis-v"></i>';
+  todoDiv.appendChild(todoDots);
+  todoList.appendChild(todoDiv);
+}
+
+function todo(event) {
+  event.preventDefault();
```
Comment (hemantso) [inline src/index.js]: - [ ] As per the learning objective we have to follow the guidelines of ES6 syntax. Kindly try to use the arrow function.

**Review by hemantso — CHANGES_REQUESTED** [review]
Hi @mariordgez, 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there! 

Alt Text
## Required Changes ♻️
_Check the comments under the review._

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (rloterh) [reply README.md]: This issue is well resolved! ✔️

File: src/index.js (hunk 2 above)
Comment (rloterh) [reply src/index.js]: This issue is well resolved! ✔️

File: src/index.js (hunk 3 above)
Comment (rloterh) [reply src/index.js]: This issue is well resolved! ✔️

File: src/index.js (hunk 4 above)
Comment (rloterh) [reply src/index.js]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Mario, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾 🍾 :shipit: .
Congratulations! 🎉 🎉 🥂

<br>
Cheers and Happy coding! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**